### DISCUSSION TITLE
Apple Add to Calendar is NOT Working with Latest iOS Version 16.1

### DISCUSSION THREAD

**Discussion by ashtonlclark** [add2cal/add-to-calendar-button#237] [Q&A] [answered]
Hello,

First and foremost, awesome add to calendar tool!

I tested the add to Apple calendar button and unfortunately it did not work with the latest version of iOS 16.1. See the video for exactly what happened  - https://share.icloud.com/photos/00fCtjZLghY6DF-uYI05HjLRw

Note: This happened on Chrome and Safari (latest versions as of this post). I assume it happens on other browsers as well. 

Any thoughts on how to fix this? Everything else worked fine.

**Comment by jekuer** [comment]
@ashtonlclark , is it the same with "Example 3" from the demo page?
And do you know if it worked with you in iOS16? I am testing with 16.0 and there it is all fine.
Thanks a lot in advance!

**Reply by ashtonlclark** [reply]
Hello @jekuer,

I just checked again. I used Example 1 from the demo page. I clicked on "Apple"

- Everything worked fine on iOS Safari
- It did not work on iOS Chrome v 107.0.5304.66 . You can see what happens via this video - https://share.icloud.com/photos/00fCtjZLghY6DF-uYI05HjLRw

I am using iOS 16.1. You can download the latest version of iOS as it should be available. I do not believe I had this issue in iOS16.

**Reply by jekuer** [reply]
Ah - so it is working in Safari, but not in Chrome, right?

**Reply by ashtonlclark** [reply]
Yes @jekuer. That is correct. The apple calendar overlay does not pop over when using chrome like it does with Safari

**Reply by jekuer** [reply]
ok. Thanks. This is a known issue. Unfortunately, we cannot make the overlay work in Chrome, but at least we have (had) a workaround to inform you about that. Seems that this is broken. I will investigate and keep you posted.

**Reply by ashtonlclark** [reply]
Yes, I just tested on my iPad which has iPad iOS 15 and I see the pop up notice. Yes, that is what is broken on the latest version of iOS.

**Comment by jekuer** [comment, marked as answer]
Reason: iOS 16 is somehow no longer supporting multiple drop-shadows in css. No idea why. I reworked the shadows (a little bit more ugly, but acceptable). It should now work again with v1.18.6

**Reply by ashtonlclark** [reply]
Looks good. Thanks for quickly addressing this fix! Nice work.
